docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: '3.4'
+
+services:
+  lts-webpack1:
+    build:
+      context: .
+      args:
+        NODE_VERSION: "lts"
+        WEBPACK_VERSION: "1"
+  lts-webpack2:
+    build:
+      context: .
+      args:
+        NODE_VERSION: "lts"
+        WEBPACK_VERSION: "2"
+  lts-webpack3:
+    build:
+      context: .
+      args:
+        NODE_VERSION: "lts"
+        WEBPACK_VERSION: "3"
+  lts-webpack4:
+    build:
+      context: .
+      args:
+        NODE_VERSION: "lts"
+        WEBPACK_VERSION: "4"
+  current-webpack1:
+    build:
+      context: .
+      args:
+        NODE_VERSION: "current"
+        WEBPACK_VERSION: "1"
+  current-webpack2:
+    build:
+      context: .
+      args:
+        NODE_VERSION: "current"
+        WEBPACK_VERSION: "2"
+  current-webpack3:
+    build:
+      context: .
+      args:
+        NODE_VERSION: "current"
+        WEBPACK_VERSION: "3"
+  current-webpack4:
+    build:
+      context: .
+      args:
+        NODE_VERSION: "current"
+        WEBPACK_VERSION: "4"
+
```

Comment:
i didn't want to push `Dockerfile` and `docker-compose.yml`. Its was only to local testing.

I started adding dependencies with aliases (webpack v1 for webpack version 1, etc.), but it did not work for `webpack` v4 because of internal dependencies that called` webpack`.